src/axom/core/tests/utils_utilities.hpp

```diff
@@ -91,16 +91,9 @@ TEST(utils_Utilities, random_real_with_seed)
   constexpr double a = -5.0;
   constexpr double b = 5.0;
 
-  const double expected_reals[5] = {-1.5112829544380526,
-                                    -2.3311429024686219,
-                                    -3.6335370551231403,
-                                    -4.714431326610093,
-                                    3.6893326916732878};
```

Comment:
Two comments here:
1. As far as I can tell, we cannot assume that different compilers will provide the same set of values for a uniform distribution when given the same seed.  See, e.g. https://stackoverflow.com/questions/28607889/how-to-generate-the-same-random-number-sequence-over-multiple-types-of-compilers
2. For some reason, the two tests in this filer were failing when I ran the `./core_serial_tests`, but not when I ran `ctest`?